config.go

```diff
@@ -1917,6 +1925,52 @@ func parseRPCParams(cConfig *lncfg.Chain, nodeConfig interface{},
 	return nil
 }
 
+// extractEnvVariable extracts the value of an environment variable from a string.
+// It supports the following formats:
+// 1) $ENVIRONMENT_VARIABLE
+// 2) ${ENVIRONMENT_VARIABLE}
+// 3) ${ENVIRONMENT_VARIABLE:-DEFAULT_VALUE}
+//
+// Standard environment variable naming conventions:
+// - ENVIRONMENT_VARIABLE contains letters, digits, and underscores, and does not start with a digit.
+// - DEFAULT_VALUE follows the rule that it can contain any characters except whitespace.
+//
+// Parameters:
+// - value: The input string containing references to environment variables.
+//
+// Returns:
+//   - string: The value of the specified environment variable, or the default value
+//     if provided, or the original input string if no matching variable is found or set.
+func extractEnvVariable(value string) string {
```

Comment:
Can you please break everything at 80 characters (while configuring your editor to use 8 spaces for tab characters).

Reply:
Thanks for letting me know! I've checked the [`code_formatting_rules.md`](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md) and configured my Visual Studio code `settings.json` to reflect the mentioned rules so that it would be reflected automatically in the next PRs:
```json
"editor.insertSpaces": false,
"editor.tabSize": 8,
"editor.wordWrap": "wordWrapColumn",
"editor.wordWrapColumn": 80,
"go.formatTool": "gofmt",
"go.useLanguageServer": true,
"editor.formatOnSave": true,
"go.lintTool": "golangci-lint",
"go.lintFlags": [
  "--fast"
],
"[markdown]": {
  "files.eol": "\n",
  "files.insertFinalNewline": true,
  "editor.wordWrap": "bounded",
  "editor.wordWrapColumn": 80
}
```